tests/scripts/thread-cert/thread_cert.py

```diff
@@ -482,15 +482,20 @@ def _parse_params(self, params: Optional[dict]) -> dict:
         params = params or {}
 
         if params.get('is_bbr') or params.get('is_otbr'):
-            # BBRs must use thread version 1.2
-            assert params.get('version', '1.2') == '1.2', params
-            params['version'] = '1.2'
+            # BBRs must not use thread version 1.1
+            version = params.get('version', '1.3')
+            assert version != '1.1', params
+            params['version'] = version
             params.setdefault('bbr_registration_jitter', config.DEFAULT_BBR_REGISTRATION_JITTER)
         elif params.get('is_host'):
             # Hosts must not specify thread version
             assert params.get('version', '') == '', params
             params['version'] = ''
 
+        # use 1.3 node for 1.2 tests
+        if params.get('version') == '1.2':
+            params['version'] = '1.3'
```

Comment:
Is it because we are not changing 'version': '1.2' in test scripts?

Reply:
Yes. I wasn't sure if we should be updating them to 1.3 given that they are really testing 1.2 functionality.

Reply:
The `version` in test topology shows what the version of the device should be. 
- 1.1 means the device should be using 1.1 firmwares
- 1.2 means the device should be using 1.2 firmwares (and also 1.3 firmware as of now)
- `not specified` means the device should use 1.1/1.2/1.3 firmwares according to the current test environment

I think for most `1.2` versions, they actually mean `1.2` or `1.3`, whichever is specified by the environment variable `THREAD_VERSION`.

I think we can do it later in a future PR.